### PR TITLE
Add kirits install script ad update Readme.

### LINKED ISSUE #89: Fix error "error listing all image policy requirements" when no image policy is configured.
Right now, when we install the Webhook without any image policies we see this error.
```
kubectl create -f exanple.yaml
Error:
 msg="fetching image security policies: error listing all image policy requirements: the server could not find the requested resource (get imagesecuritypolicies.kritis.grafeas.io)"
```
This error disappears when we deploy a image policy spec.
Lets not return false but just return true and log message "No Image policy CRD defined" or ship one by default

### PR DESCRIPTION
**Pull request by tejal29**
dupe of #94 
fixes #89

### CODE REVIEW

File: kritis-charts/README.md [hunk 1]
```diff
@@ -116,33 +116,12 @@ You will need [helm](https://docs.helm.sh/using_helm/) installed to install krit
...
-   
-   Note: Please install in the same namespace that you created the secret using the `certgen` plugin.
+2. Now run the install script.
+   This should create TLS certs, deploy kritis in given namespace.
+
    ```
-   $ helm install ./kritis-charts --namespace <your namesapce> \
-        --set caBundle=$(kubectl get secret tls-webhook-secret -o jsonpath='{.data.cert}') \
-        --set secret.name=<your secret name>
+    ./install/install-kritis.sh -n <your namespace DEFAUT=default> -s <your secret name | DEFAULT=gac-secret
```
Comment (priyawadhwa) [inline kritis-charts/README.md]: nit: <your namespace | DEFAULT=default>

File: install/install-kritis.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# Command line Args Default
+NAMESPACE="default"
+GAC_SECRET="gac-secret"
```
Comment (priyawadhwa) [inline install/install-kritis.sh]: If we're assuming the default secret name is gac-secret, we should change it in [values.yaml](https://github.com/grafeas/kritis/blob/master/kritis-charts/values.yaml#L49) and in the README, where we create secrets.

Comment (tejal29) [reply install/install-kritis.sh]: yeah we can do that. Will make those changes.

**Comment by aaron-prindle** [comment]
kokoro didn't run the integration tests here, not sure why

**Pull request merged**